build-system/tasks/release/index.js

```diff
@@ -34,7 +34,7 @@ const BASE_FLAVOR_CONFIG = {
   name: 'base',
   rtvPrefixes: ['00', '01', '02', '03', '04', '05'],
   // TODO(#28168, erwinmombay): relace with single `--module --nomodule` command.
-  command: 'gulp dist --noconfig --esm && gulp dist --noconfig',
+  command: 'gulp dist --noconfig --esm && gulp dist --noconfig && gulp dist --noconfig --sxg',
```

Comment:
Let's flip the order so that --sxg runs first, this way if there are any --esm or base-build files that would override they would take priority for now!

Reply:
Done.